Composer/packages/client/src/pages/design/exportSkillModal/index.tsx

```diff
@@ -71,6 +89,84 @@ const ExportSkillModal: React.FC<ExportSkillModalProps> = ({ onSubmit, onDismiss
     !isAdaptive ? skillConfiguration?.allowedCallers : runtimeSettings?.skills?.allowedCallers ?? []
   );
 
+  const [isCreateProfileFromSkill, setIsCreateProfileFromSkill] = useState(false);
+  let publishUpdater: PublishStatusPollingUpdater;
+  const publishNotificationRef = useRef<Notification>();
+  // stop polling updater & delete pending notification
+  const stopUpdater = async (updater) => {
+    updater.stop();
+
+    const notification = publishNotificationRef.current;
+    notification && (await deleteNotification(notification.id));
+    publishNotificationRef.current = undefined;
+    handleDismiss();
+    setIsHidden(false);
+  };
+  const changeNotificationStatus = async (data) => {
+    const { apiResponse } = data;
+    if (!apiResponse) {
+      stopUpdater(publishUpdater);
+      return;
+    }
+    const responseData = apiResponse.data;
+
+    if (responseData.status !== ApiStatus.Publishing) {
+      // Show result notifications
+      const displayedNotification = publishNotificationRef.current;
+      if (displayedNotification) {
+        const currentTarget = publishTargets?.find((target) => target.name === publishUpdater.getPublishTargetName());
+        const url = currentTarget
+          ? `https://${JSON.parse(currentTarget.configuration).hostname}.azurewebsites.net/manifests/${
```

Comment:
does this have to be hard-coded

Reply:
Yep.

Reply:
after checking with @luhan2017  and @alanlong9278 , this url is a pattern in azure that we are replying on in multiple places. It should be safe to infer endpoint based on hostname in azure environment.

Reply:
FYI, @cwhitten, i think we can keep it as for now, until we are going to support different cloud provider and in that time we will have to update more places than just this one

Reply:
even if that's the case it better to be encapsulated in a function so it's easier for extension in the future.

Reply:
Sure, I will wrap it into a function.